src/MainWizard.cpp

```diff
@@ -275,15 +275,16 @@ void MainWizard::setCinderLocationByIndex( int index )
 {
 	mCinderLocationIndex = index;
 
-	auto cinderLocationPath = Preferences::getCinderVersions()[index].path;
+    auto const cinderLocationPath = Preferences::getCinderVersions()[index].path;
+    auto const blocksPath = cinderLocationPath + "/blocks";
```

Comment:
What do you thing about using QDir::separator() instead of slash in QString variable?
`auto const blocksPath = cinderLocationPath + QDir::separator() + "blocks";`